openfisca_aotearoa/parameters/social_security/supported_living_payment/base.yaml

```diff
@@ -0,0 +1,827 @@
+description: Supported Living Payment cut-out points
```

Comment:
Thank you so much for the research gone into this parameter set, it looks perfect!